src/test/java/com/hocs/test/glue/dcu/DataInputStepDefs.java

```diff
@@ -163,7 +102,7 @@ public void theSubmittedCorrespondentShouldBeVisibleInTheListOfCorrespondents()
 
     @And("I remove the primary correspondent")
     public void removePrimaryCorrespondent() {
-        correspondents.removePrimaryCorrespondent();
+        correspondents.removeACorrespondent();
```

Comment:
This seems like a step backwards in terms of readability, does this method make sure that correspondent being removed is the primary one?